files/pt-br/web/javascript/reference/global_objects/function/arguments/index.md

```diff
@@ -45,12 +45,7 @@ console.log("returned: " + g.arguments);
 
 ## Especificações
 
-| Especificação                                                        | Status               | Comentário                                                                                                                     |
-| -------------------------------------------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
-| {{SpecName('ES1')}}                                                  | {{Spec2('ES1')}}     | Definiçao inicial. Implementado em JavaScript 1.0. Obsoleto em favor de {{jsxref("Functions/arguments", "arguments")}} em ES3. |
-| {{SpecName('ES5.1', '#sec-10.6', 'arguments object')}}               | {{Spec2('ES5.1')}}   | {{jsxref("Functions/arguments", "arguments")}} object                                                                          |
-| {{SpecName('ES6', '#sec-arguments-object', 'arguments object')}}     | {{Spec2('ES6')}}     | {{jsxref("Functions/arguments", "arguments")}} object                                                                          |
-| {{SpecName('ESDraft', '#sec-arguments-object', 'arguments object')}} | {{Spec2('ESDraft')}} | {{jsxref("Functions/arguments", "arguments")}} object                                                                          |
+Not part of any standard.
```

Comment:
It's better to localize this, but I can't speak brazilian portuguese. Feel free to modify this.